src/sbvr-api/sbvr-utils.coffee

```diff
@@ -540,14 +541,34 @@ exports.runRule = do ->
 		.nodeify(callback)
 
 exports.PinejsClient =
-	class PinejsClient extends PinejsClientCore(_, Promise)
-		_request: ({ method, url, body, tx, req, custom }) ->
+	class PinejsClient extends PinejsClientCore(_, Promise, mimemessage)
+		compileBatch: (data) -> _.map(data, batchify)
```

Comment:
I think it'd be worth adding a comment here that this is an optimisation to avoid the need to compile/parse the batch request (at least I assume that's why it's done?)